_episodes/02-installation.md

```diff
@@ -221,19 +222,16 @@ installed in it.
 >         the moment, please try again later or [raise an
```

Comment:
Does it take long now? It was quick for me and I am not aware of any issues. Do you know of any causing delays at this time?

Reply:
That was the old list. I deleted it and add a link to the documentation.